sfs/mono/source.py

```diff
@@ -82,7 +82,9 @@ def point(omega, x0, grid, c=None):
     grid = util.as_xyz_components(grid)
 
     r = np.linalg.norm(grid - x0)
-    return 1 / (4*np.pi) * np.exp(-1j * k * r) / r
+    # `r` can get `0`, which is fine in this case
```

Comment:
I don't really like the backticks here, because they are not used for actual formatting.

"which is fine in this case" sounds a bit strange to me. Who says it's fine?

The fact that "we allow it" or "it's fine" is already explicit because we use this "ignore" thing, isn't it?

What about:

```python
# If r is 0, the sound pressure is complex infinity
```

or something like that?

This explains the problem, the solution is already visible and needs no further comment.

We could (should?) probably also mention this in the docstring, right?

Reply:
Is complex infinity a defined term?
When I calculate the pressure of a point source at its source position, I'm getting a mixture of `nan` and `inf`:
```python
>>> sfs.mono.source.point(omega, x0, x0)
(inf+nanj)
```

Reply:
I'm not an expert, but I think that's NumPy's definition of "complex infinity". I couldn't find any mention of this in the NumPy docs, though.

It is a "defined term", but it's surprisingly hard to find information about it. Here's one example: http://oeis.org/wiki/Division_by_zero

FWIW, the Wolfram language has this as a separate symbol: https://reference.wolfram.com/language/ref/ComplexInfinity.html

Reply:
Thanks, for looking it up. Changed it to your proposal.